openmdao/components/tests/test_external_code_comp.py

```diff
@@ -612,6 +608,7 @@ def solve_nonlinear(self, inputs, outputs):
 
         prob.setup()
 
+        prob.set_val('area_ratio', 0.5)
```

Comment:
Is this set intentional since you override it 3 lines later?

Reply:
That was the old default value set in the IVC, but it is redundant and this is an example, so removed.